PHPCompatibility/Tests/ParameterValues/NewIDNVariantDefaultUnitTest.inc

```diff
@@ -11,3 +11,4 @@ echo idn_to_ascii('täst.de');
 echo idn_to_ascii( $domain, IDNA_DEFAULT);
 echo IDN_to_utf8('xn--tst-qla.de');
 echo idn_to_utf8( $domain, IDNA_DEFAULT);
+echo idn_to_ascii(idna_info: $idna_info, domain: 'täst.de');
```

Comment:
Are we missing a test on idn_to_utf8 for named parameters ?

Reply:
Not necessarily. The principle is being tested and the logic covered, so an additional test is not strictly necessary, though I'd be happy to add it. (for this sniff as it's only one extra test, not so much for some of the other sniffs where we'd be talking dozens of tests)